rt/databinding/jaxb/src/test/java/org/apache/cxf/jaxb/JAXBDataBindingTest.java

```diff
@@ -226,29 +225,21 @@ JAXBDataBinding createJaxbContext(boolean internal) throws Exception {
         classes.add(QualifiedBean.class);
 
         //have to fastboot to avoid conflicts of generated accessors
-        System.setProperty("com.sun.xml.bind.v2.runtime.JAXBContextImpl.fastBoot", "true");
-        System.setProperty("com.sun.xml.internal.bind.v2.runtime.JAXBContextImpl.fastBoot", "true");
-        if (internal) {
-            System.setProperty(JAXBContext.JAXB_CONTEXT_FACTORY, "com.sun.xml.internal.bind.v2.ContextFactory");
-            db.setContext(db.createJAXBContext(classes));
-            System.clearProperty(JAXBContext.JAXB_CONTEXT_FACTORY);
-        } else {
-            db.setContext(db.createJAXBContext(classes));
-        }
-        System.clearProperty("com.sun.xml.bind.v2.runtime.JAXBContextImpl.fastBoot");
-        System.clearProperty("com.sun.xml.internal.bind.v2.runtime.JAXBContextImpl.fastBoot");
+        System.setProperty("org.glassfish.jaxb.v2.runtime.JAXBContextImpl.fastBoot", "true");
+        db.setContext(db.createJAXBContext(classes));
+        System.clearProperty("org.glassfish.jaxb.v2.runtime.JAXBContextImpl.fastBoot");
         return db;
     }
 
     void doNamespaceMappingTest(boolean internal, boolean asm) throws Exception {
-        if (internal) {
+        /*if (internal) {
```

Comment:
May be we could add `TODO: ` item here, it will be easy to forget such commented out places, what do you think?

Reply:
I agree. I need to review this later to see if we can remove these checks for JDK's internal jaxb implementation.